sunpy/image/tests/test_coalignment.py

```diff
@@ -52,10 +52,10 @@ def aia171_test_template(aia171_test_shift,
                          aia171_test_map_layer,
                          aia171_test_map_layer_shape):
     # Test template
-    a1 = aia171_test_shift[0] + aia171_test_map_layer_shape[0] / 4
-    a2 = aia171_test_shift[0] + 3 * aia171_test_map_layer_shape[0] / 4
-    b1 = aia171_test_shift[1] + aia171_test_map_layer_shape[1] / 4
-    b2 = aia171_test_shift[1] + 3 * aia171_test_map_layer_shape[1] / 4
+    a1 = int(aia171_test_shift[0] + aia171_test_map_layer_shape[0] / 4)
+    a2 = int(aia171_test_shift[0] + 3 * aia171_test_map_layer_shape[0] / 4)
+    b1 = int(aia171_test_shift[1] + aia171_test_map_layer_shape[1] / 4)
+    b2 = int(aia171_test_shift[1] + 3 * aia171_test_map_layer_shape[1] / 4)
```

Comment:
seems right i will make changes accordingly asap. Thanks for suggestion, i almost forgot about that `//` operator.